### ISSUE TITLE
cant authenticate

### ISSUE THREAD

**Issue by codyc4321** [itsjafer/schwab-api#10]
Hi, I downloaded symantec VIP and told schwab about it. i dont understand where I get the topt token from

**Comment by itsjafer** [comment]
Hi. Please follow the instructions at https://github.com/itsjafer/schwab-api#create-a-totp-authentication-token to get a:

* Symantec ID to give Schwab
* TOTP token to give the API

**Comment by dougbrowning** [comment]
I figured out the code comes from Google Authenticator not VIP at all.  Did not need VIP.  Directions hard to follow on that part.
thanks
